content/source/docs/enterprise/install/installer.html.md

```diff
@@ -182,6 +185,33 @@ ADD example-intermediate-ca.crt /usr/local/share/ca-certificates/
 RUN update-ca-certificates
 ```
 
+#### RHEL 7
+
+```
+FROM registry.access.redhat.com/rhel7
+
+# running 'subscription-manager' is necessary to be able to install additional packages via yum
+# substitute your redhat account credentials in this command
+RUN subscription-manager register --username ${RH_username} --password "${RH_password}" --auto-attach
+
+# Include all necessary CA certificates.
+ADD example-root-ca.crt /usr/share/pki/ca-trust-source/anchors
+ADD example-intermediate-ca.crt /usr/share/pki/ca-trust-source/anchors
+
+# Update the CA certificates bundle to include newly added CA certificates.
+RUN update-ca-trust
+
+# Install software used by Terraform Enterprise.
+RUN yum -y install unzip sudo git openssh wget curl psmisc iproute nmap-ncat openssl rpm-build redhat-rpm-config make gcc && \
+  yum clean all && \
+  cd /tmp && git clone https://github.com/jacobm3/daemontools-rpm.git && \
+  cd /tmp/daemontools-rpm && wget https://cr.yp.to/daemontools/daemontools-0.76.tar.gz && ./buildrpm.sh && \
+  rpm -ivh /root/rpmbuild/RPMS/x86_64/daemontools-0.76-1.el7.x86_64.rpm
+```
+
+Note that the `daemontools` package is required, as it provides the `envdir` binary, which is used by TFE. If this package is not
```

Comment:
This implies to me that some of the things in here are not required? Should we tell folks which things they need and which ones they don't?  Regardless, I think I'd move this to be above the actual script, right under the heading. I worry that folks won't see it way down here :-)

Reply:
I'll move this up and reword it a bit. There is another section under the requirements that says the installed packages listed in these examples are required.